src/entities/Event/types.ts

```diff
@@ -232,3 +234,5 @@ export enum ToggleItemsValue {
   One = "one",
   Recurrent = "recurrent",
 }
+
+export const MAX_TAG_ALLOWED = 1
```

Comment:
we should use `category` instead of `tags`

```suggestion
export const MAX_CATAGORIES_ALLOWED = 1
```